src/operations/search/query/customQueries.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * This file defines the custom query filters
+ * The format is we specify the filter passed in the query.  then we define the resourceType for it and mappings that
+ *  map the resource we're searching for that filter and the reference property in that resource to use
+ */
+const customQueries = {
+    'patient': {
+        'resourceType': 'Patient',
+        'mappings': {
+            'Patient': 'id',
+            'AllergyIntolerance': 'patient.reference',
```

Comment:
Do we know if there's a case that the property value 'patient.reference' would change? If that's the case, should we create a reusable const variable ex. patientReference='patient.reference'

Reply:
this is from FHIR so it should not change